### PR TITLE
Add video attribute

### PR DESCRIPTION
**Pull request by angeli-mercedes**
- Added a video attribute to CMS
- Accepts "video/mp4" file type
- User can now add videos and edit / update the file
- Sample video: https://s3.amazonaws.com/geli-cms/7ce47fd8_mci-video.mp4

Screenshot 2023-06-08 at 2 19 02 PM

https://github.com/klaudsol/klaudsol-cms/assets/101687552/dd764410-1d0c-4274-8fa6-a0095128eb74

https://github.com/klaudsol/klaudsol-cms/assets/101687552/4c484924-b7f2-4e71-91f9-8ae6640b2bd3

Sample API:

Screenshot 2023-06-08 at 2 14 47 PM

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
     "@fortawesome/free-solid-svg-icons": "^6.1.1",
     "@fortawesome/react-fontawesome": "^0.1.18",
     "@klaudsol/commons": "https://github.com/klaudsol/klaudsol-commons.git#v1.8.0",
-    "@next/font": "^13.4.4",
+    "@next/font": "13.1.1",
```
Comment (ardeearam) [inline package.json:29]: Can we keep the version to 13.4.4?

File: package.json [hunk 2]
```diff
@@ -49,7 +49,7 @@
     "moment": "^2.29.3",
     "moment-timezone": "^0.5.34",
     "multer": "^1.4.5-lts.1",
-    "next": "^13.4.4",
+    "next": "13.1.1",
```
Comment (ardeearam) [inline package.json:52]: Can we keep the version to 13.4.4?

File: yarn.lock [hunk 3]
```diff
@@ -1053,10 +1053,10 @@
     jsonwebtoken "^9.0.0"
     multer "^1.4.5-lts.1"
 
-"@next/env@13.4.4":
-  version "13.4.4"
-  resolved "https://registry.yarnpkg.com/@next/env/-/env-13.4.4.tgz#46b620f6bef97fe67a1566bf570dbb791d40c50a"
-  integrity sha512-q/y7VZj/9YpgzDe64Zi6rY1xPizx80JjlU2BTevlajtaE3w1LqweH1gGgxou2N7hdFosXHjGrI4OUvtFXXhGLg==
+"@next/env@13.1.1":
```
Comment (ardeearam) [inline yarn.lock:1056]: Is there a reason we are downgrading?

File: package.json (hunk 2 above)
Comment (angeli-mercedes) [reply package.json:52]: I've been experiencing font issues if the next versions > 13.1.1. I researched  over the internet and people are also experiencing the same issue. https://github.com/vercel/next.js/issues/45052 

This is a sample screenshot if the version is > 13.1.1

Screenshot 2023-06-09 at 8 59 51 AM

My node version is 20.3.0. I also tried 16.18.0 and 18.13.0, but the result is the same if the version is 13.4.4.

**Pull request merged**